xray/audit/jas/iacscanner_test.go

```diff
@@ -1,28 +1,22 @@
 package jas
 
 import (
-	rtutils "github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
-	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
-	"github.com/stretchr/testify/assert"
 	"os"
 	"path/filepath"
 	"testing"
+
+	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
+	"github.com/jfrog/jfrog-cli-core/v2/xray/utils"
+	"github.com/stretchr/testify/assert"
 )
 
 func TestNewIacScanManager(t *testing.T) {
+	scanner, cleanUp := initJasTest(t, "currentDir")
+	defer cleanUp()
 	// Act
-	assert.NoError(t, rtutils.DownloadAnalyzerManagerIfNeeded())
-	scanner, err := NewAdvancedSecurityScanner([]string{"currentDir"}, &fakeServerDetails)
-	assert.NoError(t, err)
-	defer func() {
-		if scanner.scannerDirCleanupFunc != nil {
-			assert.NoError(t, scanner.scannerDirCleanupFunc())
-		}
-	}()
 	iacScanManager := newIacScanManager(scanner)
 
 	// Assert
-	assert.NoError(t, err)
 	assert.NotEmpty(t, iacScanManager)
```

Comment:
if assert.NotNill(t, iacScanManager) {
         assert.NotEmpty(t, iacScanManager.scanner.configFileName)
	assert.NotEmpty(t, iacScanManager.scanner.resultsFileName)
	assert.NotEmpty(t, iacScanManager.scanner.workingDirs)
}

so it won't throw ugly panic